### ISSUE TITLE
Object.assign isn't supported in IE but the plugin does not reflect that

### ISSUE THREAD

**Issue by benlakey** [amilajack/eslint-plugin-compat#89]
The feature "Object.assign" is not supported in any version of IE:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

The plugin however is not reporting a problem when using Object.assign.

package.json:

```
  "browserslist": [
    "last 2 ie versions"
  ]
```

eslint config:

```
  "plugins": ["compat"],
  "env": {
    "browser": true
  }
```

**Comment by dmethvin** [comment]
Edge and IE are different:

https://github.com/ai/browserslist#browsers

**Comment by benlakey** [comment]
Sorry, I updated the list of browsers in my report. The issue is still present there.

**Comment by benlakey** [comment]
I will create an example repo to highlight the issue

**Comment by benlakey** [comment]
Here you go:
https://github.com/benlakey/eslint-plugin-compat-demo

**Comment by dmethvin** [comment]
Oh, see #77. I should have noticed that but I got focused on the IE/Edge part in the original report.

**Comment by amilajack** [comment]
https://github.com/amilajack/eslint-plugin-compat/issues/72#issuecomment-288834016
